AdapterInterface.php

```diff
@@ -75,7 +75,7 @@ public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentO
      *
      * @return null|Symfony\Cmf\Component\Routing\RouteObjectInterface
      */
-    public function findRouteForUri($uri);
+    public function findRouteForUri($uri, UriContext $uriContext);
```

Comment:
I remember why I did this now .. the URI is not always the same as the `$uriContext` URI, this method is used to find candidate URI's too (e.g. conflict resolution)